spec/requests/request_spec.rb

```diff
@@ -373,9 +373,10 @@
     it 'are configured with a tooltip for removing the book format facet parameter' do
       stub_holding_locations
       get '/?f%5Bformat%5D%5B%5D=Book&q=&search_field=all_fields'
-      expect(response.body.include?('<i class="fa fa-times" aria-hidden="true" data-toggle="tooltip" data-original-title="Remove"></i>')).to eq true
```

Comment:
Not changing the expectations here, just trying to make this test a little clearer.